sdk/src/account.rs

```diff
@@ -69,6 +75,9 @@ impl Account {
             ..Self::default()
         }
     }
```

Comment:
There `new_ref` functions are mainly used in tests, could be moved someplace else?